impit-node/package.json

```diff
@@ -39,14 +39,6 @@
     "universal": "napi universal",
     "version": "napi version"
   },
-  "packageManager": "yarn@1.22.22",
-  "description": "Impit for JavaScript",
-  "optionalDependencies": {
-    "impit-darwin-arm64": "0.1.5",
-    "impit-darwin-x64": "0.1.5",
-    "impit-linux-x64-gnu": "0.1.5",
-    "impit-linux-x64-musl": "0.1.5",
-    "impit-win32-arm64-msvc": "0.1.5",
-    "impit-win32-x64-msvc": "0.1.5"
-  }
```

Comment:
so this is no longer needed? 👀

Reply:
Nope, it still works like that (will break with `npm i --no-optional`), this might be just some random autoformatting.

During (pre)publish, this part gets generated anew with the correct version numbers / target triples. It's still part of the package.json that's published on npm.